### PR TITLE
Refactor: duplicate code in resultUrls

### PR DESCRIPTION
**Pull request by sumeetrohra**
ISSUES CLOSED: #2640

### Your checklist for a pull request to sitespeed.io
Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [ ] I'm making a big change or adding functionality so I've already opened an issue proposing the change to other contributors, so I got feedback on the idea before took the time to write precious code
- [x] Check that your change/fix has corresponding unit tests (if applicable)
- [x] Squash commits so it looks sane
- [ ] Update the documentation https://github.com/sitespeedio/sitespeed.io/tree/master/docs in another PR
- [x] Verify that the test works by running `npm test` and test linting by `npm run lint`

### Description
Please describe your pull request and tell us the fix #

Thank you for making sitespeed.io even better!

### CODE REVIEW

File: lib/core/resultsStorage/resultUrls.js [hunk 1]
```diff
@@ -3,6 +3,13 @@
 const urlParser = require('url');
 const pathToFolder = require('./pathToFolder');
 
+function getPageUrl({ url, resultBaseUrl, useHash }) {
+  const pageUrl = urlParser.parse(resultBaseUrl);
+  pageUrl.pathname = [pageUrl.pathname, pathToFolder(url, useHash)].join('/');
+  console.log(urlParser.format(pageUrl));
```
Comment (soulgalore) [inline lib/core/resultsStorage/resultUrls.js]: Remove the console log :)

Comment (sumeetrohra) [reply lib/core/resultsStorage/resultUrls.js]: :+1:

**Comment by sumeetrohra** [comment]
@soulgalore made changes

**Comment by soulgalore** [comment]
Nice find @sumeetrohra ! I wonder why we have had the two from the beginning, since they do the same? :)

**Pull request merged**